src/Core/Cache/AbstractCache.php

```diff
@@ -11,6 +11,20 @@
  */
 abstract class AbstractCache implements ICache
 {
+	const TYPE_APCU      = 'apcu';
+	const TYPE_ARRAY     = 'array';
+	const TYPE_DATABASE  = 'database';
+	const TYPE_MEMCACHE  = 'memcache';
+	const TYPE_MEMCACHED = 'memcached';
+	const TYPE_REDIS     = 'redis';
+
+	/**
+	 * Force each Cache implementation to define the ToString method
+	 *
+	 * @return string
+	 */
+	abstract function __toString();
```

Comment:
I'd rather something like `getName`, I don't believe these objects have any business being cast to string.

Reply:
hm .. you're probably right .. renamed it :-)